src/axolotl/utils/config/models/input/v0_4_1/__init__.py

```diff
@@ -293,7 +293,7 @@ class LoraConfig(BaseModel):
     gptq: Optional[bool] = None
     bnb_config_kwargs: Optional[Dict[str, Any]] = None
 
-    loraplus_lr_ratio: Optional[float] = Field(
+    loraplus__ratio: Optional[float] = Field(
```

Comment:
unintentional change?

Reply:
ah yes, unintentional indeed. Thank you for catching it